ojd_daps_skills/pipeline/extract_skills/extract_skills.py

```diff
@@ -365,22 +384,30 @@ def map_skills(self, predicted_skills):
 
         return job_skills_matched_formatted
 
-    def extract_skills(self, job_adverts, map_to_tax=True):
+    def extract_skills(
+        self, job_adverts_skills: Union[str, List[dict]], format_skills=False
```

Comment:
should this be `Union[str, List[str]]`  - a single job advert, a list of job adverts or a list of skills?

Reply:
oh ya - good point !